lms/djangoapps/mobile_api/course_info/views.py

```diff
@@ -83,12 +84,10 @@ def list(self, request, course, *args, **kwargs):
                 handouts_html = None
             else:
                 handouts_html = course_handouts_module.data
-                handouts_html = replace_static_urls(
-                    handouts_html,
-                    course_id=course.id,
-                    static_asset_path=course.static_asset_path
-                )
-                handouts_html = make_static_urls_absolute(self.request, handouts_html)
+                frag = Fragment(content=unicode(handouts_html))
+                for wrapper in course_handouts_module.system.wrappers:
```

Comment:
Given this change in strategy, did you double check this new implementation against the Android simulator?